bot/exts/utils/snekio.py

```diff
@@ -0,0 +1,64 @@
+"""I/O File protocols for snekbox."""
+from __future__ import annotations
+
+from base64 import b64decode, b64encode
+from dataclasses import dataclass
+from io import BytesIO
+from pathlib import Path
+
+from discord import File
+
+# Note discord upload limit is 8 MB, or 50 MB for lvl 2 boosted servers
```

Comment:
Should be MiB, not MB

Reply:
Fixed in [daf92c6](https://github.com/python-discord/bot/pull/2326/commits/daf92c63d23ed3d4664c9003ae59d0aa4965e00a)